node/sc/multi_bridge_test.go

```diff
@@ -149,3 +149,31 @@ func TestRegisterDeregisterToken(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, common.Address{0}, res)
 }
+
+// TestOperatorThreshold checks the following:
+// - the bridge contract method SetOperatorThreshold
+func TestOperatorThreshold(t *testing.T) {
+	info := prepareMultiBridgeTest(t)
+	const vtThreshold = uint8(128)
+	const confThreshold = uint8(255)
+
+	opts := &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
```

Comment:
If possible, replace `opts` with `info.acc`. :)

Reply:
It is not working now. Let's investigate it later.

Reply:
Thanks. I will also look one more. :)